monkey/monkey_island/cc/resources/pba_file_upload.py

```diff
@@ -0,0 +1,72 @@
+import flask_restful
+from flask import request, send_from_directory, Response
+from cc.services.config import ConfigService, WINDOWS_PBA_INFO, LINUX_PBA_INFO
+import os
+from werkzeug.utils import secure_filename
+import logging
+import copy
+
+LOG = logging.getLogger(__name__)
+UPLOADS_DIR = "./monkey_island/cc/userUploads"
```

Comment:
I reference the same dir in multiple files (different relative paths) . Any way I could reference this dir once?

Reply:
Why different relative path? just make all paths relative to the island dir regardless of your current file location